### DISCUSSION TITLE
How do I send a request within a custom module

### DISCUSSION THREAD

**Discussion by atonemi** [nuxt/nuxt#24619] [Questions]
Hello everyone, I have a question I'd like to ask. I'm trying to send a request within the setup of a custom module. I initially attempted to use $fetch and useFetch, but encountered issues. So, I installed axios. However, when I use axios to request a test endpoint in my development environment (http://127.0.0.1:1789/xxx/xxx), I keep getting a 502 error. I've confirmed that the endpoint works fine in Postman and browsers. I'm not sure what to do next, and I would greatly appreciate any guidance. Thank you so much!
![image](https://github.com/nuxt/nuxt/assets/75244754/a09cccc2-30a6-4a3c-83e1-71ad12f4aed0)

**Comment by nickmadev** [comment]
@yapengsoft 
I am sorry to hear your trouble in Nuxt.js project.
Do you have any Errors in browser console like CORS problems?
If you encountered CORS problems, you can try 2 ways.
First approach is to check if ssr option is true in nuxt.config.js file.
In Nuxt.js project, ssr option is used to enable ssr feature, if ssr is true, you can avoid CORS problem.
Second approach is to add `Access-Control-Allow-Origin: *` in your header of API.
If you can't resolve with above 2 ways, please provide more context about your trouble.

**Reply by atonemi** [reply]
Thank you for your response. I've confirmed it's not a cross-origin issue; it seems to be a proxy problem. Here's the debugging information:
![image](https://github.com/nuxt/nuxt/assets/75244754/17705a44-6e0a-42e0-99c2-dc9f149ac28e)

**Reply by atonemi** [reply]
Hello, I've resolved the issue. Thank you for your response.
![image](https://github.com/nuxt/nuxt/assets/75244754/1ba88f39-5172-4b70-b360-9640faac63eb)

**Reply by nickmadev** [reply]
Okay, sounds good.
Nice job!
